components/layout_2020/fragment_tree/fragment.rs

```diff
@@ -30,7 +30,7 @@ pub(crate) enum Fragment {
     /// Floating content. A floated fragment is very similar to a normal
     /// [BoxFragment] but it isn't positioned using normal in block flow
     /// positioning rules (margin collapse, etc). Instead, they are laid out by
-    /// the [SequentialLayoutState] of their float containing block formatting
+    /// the [crate::flow::float::SequentialLayoutState] of their float containing block formatting
     /// context.
```

Comment:
Please balance these lines, the former is now considerably longer than the others in this paragraph, and the second has a single word.

Reply:
I have updated the PR @Loirooriol 
Thank you.